docs/pages/build/building-from-github.mdx

```diff
@@ -16,7 +16,14 @@ This guide explains how to trigger builds directly from your GitHub repository u
 
 To trigger EAS builds from a GitHub repo, you'll need to configure your project for EAS Build and successfully run a build from your computer for each platform that you'd like to support on GitHub.
 
-If you have run `eas build -p [all|ios|android]` successfully before, then you can continue. If you haven't done this yet, refer to the [Create your first build](setup.mdx) and return here when ready.
+If you have run `eas build -p [all|ios|android]` successfully before, then you can continue. If you
+haven't done this yet, refer to the [Create your first build](setup.mdx) and return here when ready.
+
+The following must also be true:
+
+- An Expo user in the organization must have a linked GitHub user with access to the target repository. Check your [user settings page](https://expo.dev/settings#connections) to make sure your GitHub user is linked.
+- The Expo account must have an active subscription for the feature preview.
+- You must accept the permissions requested by the [Expo GitHub app](https://github.com/settings/installations).
```

Comment:
```suggestion
If you haven't successfully run `eas build -p [all|ios|android]` yet, see  [Create your first build](/build/setup/) for more information. Once you have, continue with the steps in this guide.

The following must also be true:

- An Expo user in the organization must have a linked GitHub user with access to the target repository. Check [User settings > Connections](https://expo.dev/settings#connections) and verify that your GitHub user account is linked.
- The Expo account must have an active subscription for the feature preview.
- You must accept the permissions requested by the [Expo GitHub app](https://github.com/settings/installations).
```

- Let's be concise in the first paragraph
- Let's use the correct setting name and reference